content/influxdb/cloud/sign-up.md

```diff
@@ -0,0 +1,297 @@
+      and follow the prompts to subscribe. After you click **Set Up Your Account**,
+      enter your credentials, and then click **Start Now**.
+      All usage charges will be paid through the subscribed GCP account.
+
+      {{%note%}}
+Currently, we do **not support** using an existing InfluxDB Cloud account to sign up for an InfluxDB Cloud plan through AWS, Microsoft, or GCP Marketplaces.
+      {{%/note%}}
+
+2. If you signed up with your email address, InfluxDB Cloud requires email verification to complete the sign up process.
+   Verify your email address by opening the email sent to the address you provided and clicking **Verify Your Email**.
+3. (If you subscribed through InfluxData) Choose your cloud provider.
+4. Select a provider and region for your {{< cloud-name >}} instance. The following are available:
+
+    {{< cloud_regions type="list" >}}
+
+    _To suggest regions to add, click **Let us know** under Regions._
+5. Enter your company name.
+6. (If you subscribed through InfluxData) Review the terms of the agreement, and then select **I have viewed and agree to InfluxDB Cloud Services Subscription Agreement and InfluxData Global Data Processing Agreement**.
+   For details on the agreements, see the [InfluxDB Cloud: Services Subscription Agreement](https://www.influxdata.com/legal/terms-of-use/) and the [InfluxData Global Data Processing Agreement](https://www.influxdata.com/legal/influxdata-global-data-processing-agreement/).
+
+7. Click **Continue**, and then choose your plan:
+   - To upgrade to a Usage-Based plan, click **Upgrade Now**, set your limits (you may opt to receive an email when your usage exceeds the amount you enter in the **Limit ($1 minimum)** field). Next, enter your payment information and billing address, and then click **Upgrade**. A Ready To Rock confirmation appears; click **Start building your team**. Your plan will be upgraded and {{< cloud-name >}} opens with a default organization and bucket (both created from your email address). To review your usage and billing details at any time, see how to [access billing details](/influxdb/cloud/account-management/billing/#access-billing-details).  
+   - To keep the free plan, click **Keep**. {{< cloud-name >}} opens with a default organization and bucket (both created from your email address). _To update organization and bucket names, see [Update an organization](/influxdb/cloud/organizations/update-org/)
+    and [Update a bucket](/influxdb/cloud/organizations/buckets/update-bucket/#update-a-buckets-name-in-the-influxdb-ui)._
+   - To upgrade to an Annual plan, click **Contact Sales**, enter your information, and then click **Send**. Our team will contact you as soon as possible.
+
+## (Optional) Download, install, and use the influx CLI
+
+To use the `influx` CLI to manage and interact with your InfluxDB Cloud instance, complete the following steps:
+
+{{< tabs-wrapper >}}
+{{% tabs %}}
+[macOS](#)
+[Linux](#)
```

Comment:
We need to add Windows instructions here. The content for this change is on master.